alt-tab-macos/ui/ThumbnailsPanel.swift

```diff
@@ -91,8 +91,9 @@ class ThumbnailsPanel: NSPanel, NSCollectionViewDataSource, NSCollectionViewDele
         }
     }
 
-    func computeThumbnails() {
-        let maxSize = NSSize(width: NSScreen.main!.frame.width * Preferences.maxScreenUsage!, height: NSScreen.main!.frame.height * Preferences.maxScreenUsage!)
+    func computeThumbnails(_ screenOverride: NSScreen? = nil) {
+        let preferredScreen = screenOverride ?? Screen.getPreferredScreen() // use optional provided screen or determine it just in time
```

Comment:
`computeThumbnails` is only called from one context, in which the parameter is provided. Thus I think you could just remove `? = nil` and line 95, to simplify